frontend/ui/src/pages/Inbox/ConversationListHeader/index.module.scss

```diff
@@ -75,7 +75,14 @@
 
 .searchIcon {
   margin-top: 7px;
+
   path {
     fill: var(--color-text-gray);
+    width: 22px;
```

Comment:
Does it also work when you attach it to the `.searchIcon` and not the path? Somehow it feels strange to me to style the path that way.

Reply:
It should be in the `.searchIcon`. Fixing that